data/strings/types_strings.txt

```diff
@@ -19967,7 +19972,7 @@
     eu = Eguzki-sorgailua
     fa = ژنراتور خورشیدی
     fi = Aurinkogeneraattori
-    fr = Générateur solaire
+    fr = Panneaux solaires
```

Comment:
fr = Centrale solaire

> Not necessarily solar panels, it can be mirrors etc... see : https://wiki.openstreetmap.org/wiki/Tag:generator:source%3Dsolar

Reply:
`Centrale` can be used for power plant not for power generator

Reply:
Ok let's keep `Panneaux solaires` for power generator, it's clearer than `Générateur solaire` for general users.
and `Centrale solaire` for power plant 👍